msg/estimator_status_flags.msg

```diff
@@ -0,0 +1,72 @@
+bool control_status_vehicle_at_rest       # 26 - true when the vehicle is at rest
+
+
+# fault status
+uint32 fault_status_changes               # number of filter fault status changes
+bool fault_status_bad_mag_x               #  0 - true if the fusion of the magnetometer X-axis has encountered a numerical error
+bool fault_status_bad_mag_y               #  1 - true if the fusion of the magnetometer Y-axis has encountered a numerical error
+bool fault_status_bad_mag_z               #  2 - true if the fusion of the magnetometer Z-axis has encountered a numerical error
+bool fault_status_bad_hdg                 #  3 - true if the fusion of the heading angle has encountered a numerical error
+bool fault_status_bad_mag_decl            #  4 - true if the fusion of the magnetic declination has encountered a numerical error
+bool fault_status_bad_airspeed            #  5 - true if fusion of the airspeed has encountered a numerical error
+bool fault_status_bad_sideslip            #  6 - true if fusion of the synthetic sideslip constraint has encountered a numerical error
+bool fault_status_bad_optflow_x           #  7 - true if fusion of the optical flow X axis has encountered a numerical error
+bool fault_status_bad_optflow_y           #  8 - true if fusion of the optical flow Y axis has encountered a numerical error
+bool fault_status_bad_vel_n               #  9 - true if fusion of the North velocity has encountered a numerical error
+bool fault_status_bad_vel_e               # 10 - true if fusion of the East velocity has encountered a numerical error
+bool fault_status_bad_vel_d               # 11 - true if fusion of the Down velocity has encountered a numerical error
+bool fault_status_bad_pos_n               # 12 - true if fusion of the North position has encountered a numerical error
+bool fault_status_bad_pos_e               # 13 - true if fusion of the East position has encountered a numerical error
+bool fault_status_bad_pos_d               # 14 - true if fusion of the Down position has encountered a numerical error
+bool fault_status_bad_acc_bias            # 15 - true if bad delta velocity bias estimates have been detected
+bool fault_status_bad_acc_vertical        # 16 - true if bad vertical accelerometer data has been detected
+bool fault_status_bad_acc_clipping        # 17 - true if delta velocity data contains clipping (asymmetric railing)
+
+
+# innovation test failures
+uint32 innovation_fault_status_changes    # number of filter fault status changes
+bool reject_hor_vel                       #  0 - true if horizontal velocity observations have been rejected
+bool reject_ver_vel                       #  1 - true if vertical velocity observations have been rejected
+bool reject_hor_pos                       #  2 - true if horizontal position observations have been rejected
+bool reject_ver_pos                       #  3 - true if true if vertical position observations have been rejected
```

Comment:
```suggestion
bool reject_ver_pos                       #  3 - true if vertical position observations have been rejected
```
:D

Reply:
Thanks, copied verbatim from ecl if that wasn't obvious. https://github.com/PX4/PX4-ECL/blob/9d22f140dd6f5efc2b873a7827588f1fa8d67074/EKF/common.h#L420